.github/workflows/deploy-gcp-tests.yml

```diff
@@ -250,6 +263,18 @@ jobs:
           --mount type=volume,src=${{ inputs.disk_prefix }}-${{ inputs.test_id }}-${{ env.GITHUB_SHA_SHORT }},dst=${{ inputs.root_state_path }}/${{ inputs.zebra_state_dir }} \
           ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:sha-${{ env.GITHUB_SHA_SHORT }}"
 
+      # Create image from disk that will be used for following tests
+      # Force the image creation as the disk is still attached even though is not being used by the container
+      - name: Create image from state disk
+        if: ${{ inputs.saves_to_disk }}
+        run: |
+          gcloud compute images create ${{ inputs.disk_prefix }}-${{ env.GITHUB_REF_SLUG_URL }}-${{ env.GITHUB_SHA_SHORT }}-v${{ env.STATE_VERSION }}-${{ env.NETWORK }}-${{ inputs.disk_suffix }} \
```

Comment:
Can we split the zebra and lightwalletd disk prefixes into separate variables?

I need the zebra disk prefix in PR #4385 to find the zebra disk for the main branch.

(And in PR #4303, I'll need the lwd disk prefix to find the lwd disk for the main branch. But we can make that change in that PR.)

Reply:
Actually, I think we'll want to split the input and output prefix.
(Sometimes the input will be zebrad, and sometimes it will be lwd.)

These changes and the new lwd disk from #3545 seem a bit separate to the main point of this PR.
What do you think, could we do them in another PR?

Reply:
I think #4269 will be the place to finalize this decisions. I do agree that this changes might be needed (as I have local changes in my computer with similar scenarios) but based on how this disk is working for the quick update sync (if it works at all) then we will be able to take a better decision

Reply:
I think working this and #4269 in the same PR is easier, and it will give us more visibility of the needed changes, as #4269 depends on this one, but the design is interdependent

Reply:
I'm not sure if we want to put both states in the same directory on the disk, or if we even can, so I'm re-opening this conversation.

Reply:
It seems like using `disk_prefix` for both input and output disks might be causing a bit of confusion and unnecessary complexity.